proto-lens-protoc/src/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -60,15 +61,21 @@ fileSyntaxType f = case f ^. syntax of
     "" -> Proto2  -- The proto compiler doesn't set syntax for proto2 files.
     s -> error $ "Unknown syntax type " ++ show s
 
+-- Whether to import the "Reexport" modules or the originals;
+-- e.g., Data.ProtoLens.Reexport.Data.Map vs Data.Map.
+data UseReexport = UseReexport | UseOriginal
```

Comment:
Switched to `ImportDecl -> ImportDecl`.  Since that type's pretty simple, I don't think it's worth adding a newtype.